Documentation/Functions/Calc.rst

```diff
@@ -18,7 +18,7 @@ Calculating values (+calc)
 
 Sometimes a data type is set to ``someType +calc``. The ``+calc`` indicates
 that the value is calculated with ``+-/\*`` operators. *Be aware that the
-operators have no "weight".* The calculation is just done from left to
+operators have no "weight".* The calculation is done from left to
 right.
```

Comment:
```suggestion
right instead.
```
(the just was in comparision to the operator weight, this meaning should be kept)

Reply:
Why "instead"? It is done from left to right.

Reply:
Yes from left to right instead of by weight (Punkt vor Strich)

Reply:
Hmm, then we should write something like "... instead of order of operations (multiplication and division before addition and subtraction)." What do you think?

Reply:
I updated the suggestion